### DISCUSSION TITLE
Schedule connection resets

### DISCUSSION THREAD

**Discussion by mariana-s-fernandes** [airbytehq/airbyte#37529] [Connector Ideas and Features]
I would like to be able to schedule connection resets. Every time I need to reset a connection due to a schema change, there is downtime and I don't want to do it during working hours. I want to be possible to schedule these resets to happen during the night so I don't have downtime when people are trying to access the data.

**Comment by nataliekwong** [comment]
HI @mariana-s-fernandes ! What destination are you using? While we don't have automation in place yet, you can now utilize our Refreshes feature which does not clear your existing data until the new data is loaded, at which point the tables are swapped. This is now available for BigQuery, Snowflake and Redshift and will be added to additional Airbyte Connector destinations in the coming weeks.

[a comment by an automated account is not shown]
